src/Components/Common/Breadcrumbs.tsx

```diff
@@ -68,7 +68,7 @@ export default function Breadcrumbs(props: any) {
               </Link>
             </div>
           </li>
-          {!showFullPath && crumbs.length > 2 && (
+          {!showFullPath && crumbs!.length > 2 && (
```

Comment:
```suggestion
          {!showFullPath && crumbs?.length > 2 && (
```

shouldn't it supposed to be `?` instead of `!`

Reply:
@khavinshankar It should be `!` not `?`. The reason is `crumbs` can possibly be undefined because in the new version of raviger, `usePath()` can return null. [see this line](https://github.com/kyeotic/raviger/blob/main/src/location.ts#L61)

Reply:
![image](https://user-images.githubusercontent.com/29787772/177545247-6d5d1723-4832-4ad8-9965-719a478b7945.png)